src/main/java/software/amazon/qldb/TransactionExecutor.java

```diff
@@ -48,11 +52,15 @@ public void abort() {
      *              The PartiQL statement to be executed against QLDB.
      *
      * @return The result of executing the statement.
-     * @throws software.amazon.awssdk.services.qldbsession.model.QldbSessionException if there is an error executing against QLDB.
+     * @throws QldbSessionV2Exception if there is an error executing against QLDB.
      */
     @Override
     public Result execute(String statement) {
-        return transaction.execute(statement);
+        try {
+            return transaction.execute(statement);
+        } catch (ExecutionException e) {
+            throw (SdkException)e.getCause();
```

Comment:
In the new MV, we shouldn't throw the SdkException. Instead let's throw a driver exception that wraps the sdk exception.